catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/DorisTableOperationsIT.java

```diff
@@ -411,4 +413,72 @@ public void testCreateNotSupportTypeTable() {
                       "Couldn't convert Gravitino type %s to Doris type", type.simpleString())));
     }
   }
+
+  @Test
+  public void testCreateTableWithPartition() {
+    String tableComment = "partition_table_comment";
+    JdbcColumn col1 =
+        JdbcColumn.builder()
+            .withName("col_1")
+            .withType(Types.IntegerType.get())
+            .withNullable(false)
+            .build();
+    JdbcColumn col2 =
+        JdbcColumn.builder().withName("col_2").withType(Types.BooleanType.get()).build();
+    JdbcColumn col3 =
+        JdbcColumn.builder().withName("col_3").withType(Types.DoubleType.get()).build();
+    JdbcColumn col4 =
+        JdbcColumn.builder()
+            .withName("col_4")
+            .withType(Types.DateType.get())
+            .withNullable(false)
+            .build();
+    List<JdbcColumn> columns = Arrays.asList(col1, col2, col3, col4);
+    Distribution distribution =
+        Distributions.hash(DEFAULT_BUCKET_SIZE, NamedReference.field("col_1"));
+    Index[] indexes = new Index[] {};
+
+    // create table with range partition
```

Comment:
It's better to add a read/write test for the partitioned table in `com.datastrato.gravitino.catalog.doris.integration.test.CatalogDorisIT`

Reply:
You mean adding integration tests for creating and loading partitioned table in `CatalogDorisIT`?

Reply:
yes

Reply:
And add read/write tests like [HiveCatalogIT](https://github.com/datastrato/gravitino/blob/f207466f143dc21cb4fd015e7b98c1ab26782025/catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/integration/test/CatalogHiveIT.java#L681)

Reply:
Because partition operations are not supported at the moment, it is impossible to test the read and write of partitioned tables. I think this makes sense.

> In addition, we would have to bypass Gravitino for tests that read and write data, e.g. using spark or JDBC, which doesn't seem to be necessary for Gravitino in my opinion.

We need to ensure that tables created through Gravitino can be read and written smoothly by other engines, so it is very necessary to test reading and writing partitioned tables when the Doris catalog supports partition operations.